src/main/java/com/mraof/minestuck/computer/editmode/DeployList.java

```diff
@@ -349,55 +363,79 @@ public static DeployEntry getEntryForName(String name, EntryLists entryList)
 		return null;
 	}
 	
-	public static DeployEntry getEntryForItem(ItemStack stack, SburbConnection c, Level level)
+	public static DeployEntry getEntryForItem(ItemStack stack, SburbPlayerData playerData, Level level)
 	{
-		return getEntryForItem(stack, c, level, EntryLists.ALL);
+		return getEntryForItem(stack, playerData, level, EntryLists.ALL);
 	}
 	
-	public static DeployEntry getEntryForItem(ItemStack stack, SburbConnection c, Level level, EntryLists entryList)
+	public static DeployEntry getEntryForItem(ItemStack stack, SburbPlayerData playerData, Level level, EntryLists entryList)
 	{
 		stack = cleanStack(stack);
 		for(DeployEntry entry : entryList.getList())
-			if(ItemStack.matches(stack, entry.getItemStack(c, level)))
+			if(ItemStack.matches(stack, entry.getItemStack(playerData, level)))
 				return entry;
 		return null;
 	}
 	
 	
 	public interface IAvailabilityCondition
 	{
-		boolean test(SburbConnection connection);
+		boolean test(SburbPlayerData playerData);
 	}
 	
 	public static IAvailabilityCondition config(ForgeConfigSpec.BooleanValue config)
 	{
-		return connection -> config.get();
+		return playerData -> config.get();
 	}
 	
-	public static BiFunction<SburbConnection, Level, ItemStack> item(ItemLike item)
+	public static BiFunction<SburbPlayerData, Level, ItemStack> item(ItemLike item)
 	{
-		return (sburbConnection, world) -> new ItemStack(item);
+		return (playerData, world) -> new ItemStack(item);
 	}
 	
-	static CompoundTag getDeployListTag(MinecraftServer server, SburbConnection c)
+	static CompoundTag getDeployListTag(MinecraftServer server, SburbPlayerData playerData)
 	{
-		return getDeployListTag(server, c, EntryLists.ALL);
+		return getDeployListTag(server, playerData, EntryLists.ALL);
 	}
 	
-	static CompoundTag getDeployListTag(MinecraftServer server, SburbConnection c, EntryLists entryList)
+	static CompoundTag getDeployListTag(MinecraftServer server, SburbPlayerData playerData, EntryLists entryList)
 	{
 		CompoundTag nbt = new CompoundTag();
 		ListTag tagList = new ListTag();
 		nbt.put("l", tagList);
-		int tier = SburbHandler.availableTier(server, c.getClientIdentifier());
+		int tier = SburbHandler.availableTier(server, playerData.playerId());
 		for(int i = 0; i < entryList.getList().size(); i++)
 		{
 			DeployEntry entry = entryList.getList().get(i);
-			entry.tryAddDeployTag(c, server.getLevel(Level.OVERWORLD), tier, tagList, i);
+			entry.tryAddDeployTag(playerData, server.getLevel(Level.OVERWORLD), tier, tagList, i);
 		}
 		return nbt;
 	}
 	
+	private static long lastDay;
+	
+	@SubscribeEvent
+	public static void serverStarting(ServerStartingEvent event)
+	{
+		ConsortDialogue.serverStarting();
```

Comment:
The method called here is in what I would have assumed is an irrelevant class and is empty